src/supporting-data/citations/components/LiteratureCitation.tsx

```diff
@@ -1,11 +1,14 @@
-import { useState, FC, Fragment } from 'react';
+import { createElement, useState, FC, Fragment, HTMLAttributes } from 'react';
```

Comment:
![image](https://user-images.githubusercontent.com/3329765/117004443-fbf7c100-acdd-11eb-9e5d-2e7b87f6456b.png)
Not sure when this was lost, but the "Cited for/Source etc.." section on the left used to be right under the "view abstract" (to gain a bit of vertical space)

Reply:
Not really corresponding to that part of the code, but I went and changed lower in this file where the child was injected.
Also, didn't really know beforehand where it was supposed to go because I didn't remember seeing it in the past 😅

Reply:
Sorry I wasn't clear (and without having seeing it before, you're coding in the dark!) - this is how it should be (still left column but right under "View abstract"
![image](https://user-images.githubusercontent.com/3329765/117112700-5e55cd80-ad81-11eb-8384-ce4f47d0e3ba.png)

Reply:
Ah I see, I put it back below the abstract then